docs/api/shell.md

```diff
@@ -58,6 +58,9 @@ Rejects if there was an error while deleting the requested item.
 This moves a path to the OS-specific trash location (Trash on macOS, Recycle
 Bin on Windows, and a desktop-environment-specific location on Linux).
 
+On Windows, this function requires that the `path` parameter must use
```

Comment:
We should instead say that users should use `path.resolve` to ensure correct handling on all filesystems.